www/front_src/src/Resources/index.test.tsx

```diff
@@ -759,8 +730,10 @@ describe(Resources, () => {
 
     const chipLabel = `${acknowledgedEntity?.name} ${labelAcknowledged}`;
 
-    fireEvent.mouseEnter(getByLabelText(chipLabel));
-    fireEvent.mouseOver(getByLabelText(chipLabel));
+    const chip = await findByLabelText(chipLabel);
+
+    fireEvent.mouseEnter(chip);
+    fireEvent.mouseOver(chip);
 
```

Comment:
performance could be also improved by using `within` on the next line
something like : 
```
const tooltip = getByRole('tooltip');
expect(within(tooltip).getByText('admin')).toBeInTheDocument();
...
```